app/utils/formatters.js

```diff
@@ -0,0 +1,26 @@
+const moment = require("moment");
+
+function isDate(input_date) {
+    return input_date && moment(input_date).isValid();
+}
+
+function formatDate(input_date) {
+    return input_date ? moment(input_date).format('YYYY-MM-DD') : input_date;
+}
+
+function formatBoolean(input) {
+    return typeof input === "boolean"
```

Comment:
is it possible to make it easier?

Reply:
Tryied another approach, really looks better, is it better?)